### PR TITLE
Update schema for UserInfo.is_internal_user

### PR DESCRIPTION
**Pull request by scottmcmaster**

### CODE REVIEW

File: proto/common/common.proto [hunk 1]
```diff
@@ -72,7 +76,7 @@ message ClientInfo {
   ClientType client_type = 1;
 
   // Used to indicate the type of traffic.  We can use this to prioritize resources.
-  // Next ID = 5.
+  // Next ID = 6.
```
Comment (jinyius) [inline proto/common/common.proto:79]: undo or set reserved...

File: proto/delivery/blender.proto [hunk 2]
```diff
@@ -6,10 +6,11 @@ option java_package = "ai.promoted.proto.delivery";
 option java_outer_classname = "Blender";
 
 // See: https://github.com/promotedai/blender for README
-// Next ID = 10.
+// Next ID = 11.
```
Comment (jinyius) [inline proto/delivery/blender.proto:9]: undo/reserved?

**Review by jinyius — APPROVED** [review]

**Review by danbosnichill — APPROVED** [review]
Thanks for updating this!

File: proto/common/common.proto (hunk 1 above)
Comment (scottmcmaster) [reply proto/common/common.proto:79]: We should start catching more of these at the point they are introduced :-(

**Pull request merged**